rcsb/workflow/wuw/WuwUtils.py

```diff
@@ -0,0 +1,21 @@
+"""Hashing functions for file storage."""
+##
+# File: hashDirectories.py
+# Date: 04-Jun-2025  mjt
+#
+#  Shared hashing function  - for file generation
+#
+#  Updates:
+#  04-Jun-2025 mjt Created hashing function
+#
+##
+__docformat__ = "google en"
+__author__ = "Michael Trumbull"
+__email__ = "michael.trumbull@rcsb.org"
+__license__ = "Apache 2.0"
+
+
+def idHash(name):
+    if "_" in name:
+        return f"{name[0:2]}/{name[-6:-4]}/{name[-4:-2]}/"
+    return f"{name[-3:-1]}/"
```

Comment:
Likewise, I think it would be prudent to check here that the name is only 4-charcters, and then add another "else" conditional raising an error if an ID doesn't fit into either of the above conditionals.

Reply:
Yes I just confirmed with Jasmine as well that that is the plan. However, the parent and subdirectory structure of the new archive will be different, so we will likely need to implement a new condition here to handle those uniquely when the time comes. It might be prudent for now to add something like:
```python
if name.lower().startswith("pdb_"):
    raise ValueError("Hashing method not implemented yet for extended PDB identifiers - %s" % name)
```